src/LibVLCSharp/Structures/MediaTrack.cs

```diff
@@ -284,7 +284,8 @@ internal MediaTrackList(IntPtr mediaTrackListPtr) : base(() => mediaTrackListPtr
         {
             get
             {
-                if (position > Count)
+                var count = Count;
+                if (count == 0 || position >= count)
```

Comment:
Could be simplified to the one-liner :
```suggestion
                if (position >= Count)
```

if `Count == 0` there's no way for an `uint` to be below

Reply:
Sure